nx/source/kernel/thread.c

```diff
@@ -24,21 +51,41 @@ Result threadCreate(
 
         if (R_SUCCEEDED(rc))
         {
-            u64 stack_top = ((u64)stack_mirror) + t->stack_sz;
+            u64 stack_top = ((u64)stack_mirror) + t->stack_sz - sizeof(ThreadEntryArgs);
```

Comment:
Must be 16-aligned

Reply:
It already is, ThreadEntryArgs has an extra member to keep 16-byte alignment.

Reply:
Yeah, but this can explode later when an innocent looking change breaks everything xD

Reply:
Should I add a comment to ThreadEntryArgs in case it needs to be changed in the future?

Reply:
Yeah, or you could add code to align